src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -17,6 +17,8 @@
     <NuGetTargetMoniker Condition="'$(NuGetTargetMoniker)' == ''">.NETStandard,Version=v1.7</NuGetTargetMoniker>
     <!-- copy to uap10.1 until nuget makes the change to map that to ns1.7 -->
     <PackageTargetFramework Condition="'$(TargetGroup)' == ''">netstandard1.7;uap10.1</PackageTargetFramework>
+    <!-- Temporarily ignore missing types for uap101aot -->
+    <GenFacadesArgs Condition="'$(TargetGroup)' == 'uap101aot'">$(GenFacadesArgs) -ignoreMissingTypes</GenFacadesArgs>
```

Comment:
Do you really need the `-ignoreMissingTypes` flag? ApiCompatBaseline.uap101aot.txt is the better way to suppress the missing types.